cql3/statements/select_statement.cc

```diff
@@ -1534,7 +1537,8 @@ parallelized_select_statement::do_execute(
         tracing::make_trace_info(state.get_trace_state()),
         query_id::create_null_id(),
         query::is_first_page::no,
-        options.get_timestamp(state)
+        options.get_timestamp(state),
+        db::allow_per_partition_rate_limit::no
```

Comment:
These changes should be dropped.

Reply:
I made a rebase and apply the changes into first commit.